codec/aminojson/Makefile

```diff
@@ -0,0 +1,18 @@
+GOLANG_PROTOBUF_VERSION = 1.28.1
+BIN = $(CURDIR)/bin
+BUF = $(BIN)/buf
+
+.PHONY: proto-gen
+
+$(BIN)/protoc-gen-go:
+	GOBIN=$(BIN) go install google.golang.org/protobuf/cmd/protoc-gen-go@v$(GOLANG_PROTOBUF_VERSION)
```

Comment:
I thought this wasn't necessary anymore with buf?

Also see comment below about using `protoc-gen-go-pulsar` instead.